src/drawbot_skia/drawing.py

```diff
@@ -42,7 +43,21 @@ def size(self, width, height):
             raise DrawbotError("size() can't be called if there's already a canvas active")
         self.newPage(width,  height)
 
-    def newPage(self, width, height):
+    def newPage(self, *dimensions):
+        assert len(dimensions) in (0, 2), (
```

Comment:
This is an api entry point, and wrong arguments should raise an proper exception. In this case: TypeError is the right exception.